packages/flutter/lib/src/material/slider.dart

```diff
@@ -158,6 +167,8 @@ class _SliderRenderObjectWidget extends LeafRenderObjectWidget {
       ..label = label
       ..activeColor = activeColor
       ..onChanged = onChanged;
+    // Ticker provider cannot change since there's a 1:1 relationship between
+    // the _SliderRenderObjectWidget object and the _SliderState object.
```

Comment:
Right now there's no way to tell -- the ticker provider isn't kept around to compare against.